docs/content/operations/api-reference.md

```diff
@@ -162,15 +162,15 @@ Returns a list of datasource names found in the cluster.
 
 * `/druid/coordinator/v1/datasources?simple`
 
-Returns a list of JSON objects containing the name and properties of datasources found in the cluster.  Properties include segment count, total segment byte size, minTime, and maxTime.
+Returns a list of JSON objects containing the name and properties of datasources found in the cluster.  Properties include segment count, total segment byte size, replicated total segment byte size, minTime and maxTime.
```

Comment:
nit: lost a comma, 
> minTime, and maxTime.

Reply:
Thanks for reviewing. It turns out both are grammatically correct: https://www.grammarly.com/blog/comma-before-and/
Regardless, I've added the comma back :)

Reply:
Heh, indeed, that's why i called it a 'nit', but oxford comma is the one true way imo! 😅 Thanks for fixing :+1: